python/orca/src/bigdl/orca/data/file.py

```diff
@@ -21,12 +21,18 @@
 import functools
```

Comment:
The function in this file better all be typed. Maybe used by users.

Reply:
done then